packages/lodestar/src/chain/blocks/verifyBlock.ts

```diff
@@ -296,8 +296,8 @@ export async function verifyBlockStateTransition(
     if (isMergeTransitionBlock) {
       const mergeBlock = block.message as bellatrix.BeaconBlock;
       const powBlockRootHex = toHexString(mergeBlock.body.executionPayload.parentHash);
-      const powBlock = await chain.eth1.getPowBlock(powBlockRootHex);
-      const powBlockParent = powBlock && (await chain.eth1.getPowBlock(powBlock.parentHash));
+      const powBlock = await chain.eth1.getPowBlock(powBlockRootHex).catch((_error) => null);
+      const powBlockParent = powBlock && (await chain.eth1.getPowBlock(powBlock.parentHash).catch((_error) => null));
```

Comment:
This is very serious error we must know about, at least log it

Reply:
If `isMergeTransitionBlock == true` and `powBlock == null` it's impossible for us to accept this block right?

Reply:
no, we can still import it if `execStatus` is `SYNCING` and we are not  `safeSlotsToImportOptimistically` (128) blocks of the head slot (i.e. merge has long happened, this is just syncing past merge even if EL doesn't has merge block), which is when this fetch actually throws error.

Another scenario is if `TERMINAL_BLOCK_HASH` is set, then also `powBlock` is not required.

`assertValidTerminalPowBlock` takes care of these checks so we can safely assign it null and let it move to validation.

Logging them is a good idea! will update :+1:

Reply:
Why not move the fn `assertValidTerminalPowBlock` into the verifyBlock.ts, and do the fetching there? Seems like things will be more straight forward to reason about.

Reply:
ummm well there are couple of considerations:
i) forkchoice spec tests have spec test
ii) the https://github.com/ChainSafe/lodestar/issues/3508 will require this validation as per spec once the merge block status changes from SYNCING to VALID. 

Lets leave it here for the moment and see if we can place it better when working on the ii) ? :thinking: